pkg/packagekit/updates.jsx

```diff
@@ -1313,7 +1298,7 @@ class OsUpdates extends React.Component {
                     this.watchUpdates(transactionPath)
                             .then(() => {
                                 PK.call(transactionPath, PK.transactionInterface, "UpdatePackages", [0, ids])
-                                        .fail(ex => {
+                                        .catch(ex => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17517-20220705-092548-cbb7d34f-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)